src/app/views/certificates/NewCertificate.js

```diff
@@ -17,7 +17,7 @@ import { Breadcrumb } from "matx";
 import StudentAutoComplete from "./certificates-utils/StudentAutoComplete";
 
 import {AutocompleteCohorts} from "../../components/AutocompleteCohorts"
-import {AutocompleteUsers} from "../../components/AutocompleteUsers"
+// import {AutocompleteUsers} from "../../components/AutocompleteUsers"
```

Comment:
remove the comment.. another thing... why are you using AutocompelteCohorts? Re-use the AsyncAutcomplete component instead.